(<class 'AssertionError'>, AssertionError(), <traceback object at 0x0000027B34084400>)